xtokens/src/lib.rs

```diff
@@ -22,28 +22,37 @@
 #![allow(clippy::unused_unit)]
 #![allow(clippy::large_enum_variant)]
 
-use frame_support::{pallet_prelude::*, traits::Get, transactional, Parameter};
+use frame_support::{
+	pallet_prelude::*,
+	storage::{with_transaction, TransactionOutcome},
+	traits::Get,
+	Parameter,
+};
 use frame_system::{ensure_signed, pallet_prelude::*};
 use sp_runtime::{
 	traits::{AtLeast32BitUnsigned, Convert, MaybeSerializeDeserialize, Member, Zero},
 	DispatchError,
 };
 use sp_std::prelude::*;
 
-use xcm::v0::{
-	Junction::*,
-	MultiAsset, MultiLocation, Order,
-	Order::*,
-	Xcm::{self, *},
-};
+use xcm::v0::prelude::*;
+use xcm_executor::traits::WeightBounds;
 
+pub use module::*;
 use orml_traits::location::{Parse, Reserve};
-use orml_xcm_support::XcmHandler;
 
-mod mock;
-mod tests;
+// mod mock;
```

Comment:
is this intended?

Reply:
Yes. A lot of work is needed on `xcm-simulator` to update mock and unit tests. It will be done in another PR.